pushed_notifications_test.py

```diff
@@ -388,13 +388,19 @@ def test_tombstone_failure_threshold_message(self):
         have_v5_protocol = self.supports_v5_protocol(self.cluster.version())
 
         self.fixture_dtest_setup.allow_log_errors = True
-        self.cluster.set_configuration_options(
-            values={
-                'tombstone_failure_threshold': 500,
-                'read_request_timeout_in_ms': 30000,  # 30 seconds
-                'range_request_timeout_in_ms': 40000
-            }
-        )
+
+        supports_guardrails = self.cluster.version() >= LooseVersion('4.0')
+        if supports_guardrails:
+            config_options = {'guardrails': {'tombstone_warn_threshold': -1,
```

Comment:
what happens if we don't put `-1` here? what's the default?

Reply:
Without this change the test fails because the default, 1000, is greater than the failure threshold of 500 that is set.